app/src/main/java/com/example/returnpals/viewmodel/LoginViewModel.kt

```diff
@@ -103,6 +109,6 @@ class LoginViewModel(
                 }
             }
         }
+        navController.goto(MenuRoutes.PickupProcess)
```

Comment:
Thank you for your input. I found this to be the start of a new process just for the user to schedule a return. What business do they need going back to the main menu screen they already visited? They could also be inputting their address information while they log in as the guest backend runs. I am not sure what services we defined for guest users but I thought they only needed to schedule a return, no dashboard or settings or profile page to worry about updating plan by email. I feel like it is more of a continuation as a guest rather than logging in.  We can define more stuff to provide guests that may require login, I'm not sure.

Reply:
I was thinking it would go to the initial page for marketing purposes and about/info of the company. Its not super important where the guest user goes after login, just preferrance.